gulpfile.cjs

```diff
@@ -69,8 +69,8 @@ if (!concurrency) {
 
 // Work-around until all third party libraries use npm
 const filesToLeaveInThirdParty = [
-  "!Source/ThirdParty/Workers/*",
-  "!Source/ThirdParty/*.wasm",
+  "!Source/ThirdParty/Workers/basis_transcoder.js",
+  "!Source/ThirdParty/basis_transcoder.wasm",
   "!Source/ThirdParty/google-earth-dbroot-parser.js",
   "!Source/ThirdParty/knockout*.js",
   "!Source/ThirdParty/measureText.js",
```

Comment:
Can basis, measureText, and Uri be removed from this list?

Reply:
This is a list of files we keep in Source/ThirdParty/ as part of our repo. I added the list once we started saving the third party libraries built from our npm imports so that when we rebuild the npm files, the ones in this list don't get erased. measureText and Uri were removed in main; I'll merge that in, but the basis files should stay because they won't get recreated during buildThirdParty until we replace them from npm modules (or something else).

Reply:
Oh whoops, I was mixing up basis and meshopt. Ignore the comment about basis.